jobs/generation/MetaGenerator.groovy

```diff
@@ -227,11 +227,13 @@ repos.each { repoInfo ->
         return;
     }
 
-    // Make the folders
+    // Make the folders. Save the root folder for overview generation
     def generatorFolder = ''
+    def projectFolder = ''
     for (folderElement in repoInfo.folders) {
         if (generatorFolder == '') {
             generatorFolder = folderElement
+            projectFolder = folderElement
```

Comment:
What about overlapping job names between folders?  What happens then?  My other concern about this is maybe what happens when explicit folders are specified in the job configurations.  This also may not work for the Private folder, as all projects under that will get rolled up.